src/data/real/ennreal.lean

```diff
@@ -748,6 +748,10 @@ section sum
 
 open finset
 
+/-- A product of finite numbers is still finite -/
+lemma prod_lt_top {s : finset α} {f : α → ennreal} (h : ∀a∈s, f a < ∞) : (∏ a in s, f a) < ∞ :=
```

Comment:
I thought that `≠ ∞` was the preferred way to write this in `ennreal`? But if it's not standard, I'm happy to accept this, and try to uniformize this in a later PR.

Reply:
In `data/real/ennreal`  many lemmas are stated both for  `< ∞` and `≠ ∞`. Probably we should uniformize this at some point.